src/ui/Avatar/Avatar.tsx

```diff
@@ -3,10 +3,7 @@ import React from 'react';
 import * as AvatarPrimitive from '@radix-ui/react-avatar';
 
 import { styled, CSS } from '../../stitches.config';
-import {
```

Comment:
Can you also create a Storybook story for Avatar that shows how this component looks both with a logo supplied, and without one.

Reply:
I still don't understand the storybook idea but I will get into it soon enough, maybe tomorrow